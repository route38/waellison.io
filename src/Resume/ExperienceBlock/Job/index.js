import React from "react";

class Job extends React.Component {
  render() {
    let startYear = this.props.startDate.split("-")[0];
    let endYear = "present";
    let duties = this.props.duties;
    let competencies = this.props.competencies;
    if (this.props.endDate != null) {
      endYear = this.props.endDate.split("-")[0];
    }

    return (
      <div className="event">
        <h3>{this.props.title}</h3>
        <p>
          {startYear} to {endYear} at {this.props.company}
        </p>
        <ul>
          {duties.map((duty, idx) => (
            <li key={idx} dangerouslySetInnerHTML={{ __html: duty }} />
          ))}
        </ul>
        <ul className="skillsUsed">
          {competencies.map((skill, idx) => (
            <li
              className="skillTag"
              key={idx}
              dangerouslySetInnerHTML={{ __html: skill }}
            />
          ))}
        </ul>
      </div>
    );
  }
}

export default Job;
