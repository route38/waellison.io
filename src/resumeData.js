const resumeData = {
  summary:
    "I am a passionate, skilled software developer with a strong background in web development and three years' experience in healthcare revenue cycle management.  My skills include programming in Python and JavaScript with the Flask and React frameworks, PostgreSQL databases, and REST APIs.  I am also skilled at developing for the cloud with Docker, Kubernetes, and Microsoft Azure.  I have recognized skills in analysis, problem solving, and written and oral communication.",
  skills: {
    DevOps: [
      "azure",
      "docker",
      "kubernetes",
      "git",
      "linux",
      "continuousIntegration",
    ],
    Libraries: ["flask", "jQuery", "reactjs", "restApi", "postgreSQL"],
    Languages: ["python", "javaScript", "ceePlusPlus", "english", "french"],
  },
  education: {
    fcc: {
      institution: "freeCodeCamp",
      credential: "Certification in Frontend Web Libraries (in-progress)",
      startDate: "2022-03-15",
      endDate: null,
      description:
        "Self-paced boot camp-style curriculum in major libraries used for frontend Web development, including jQuery, React, Redux, and Bootstrap.",
      competencies: [
        "JavaScript",
        "React.js",
        "Redux",
        "HTML",
        "CSS",
        "Bootstrap",
      ],
    },
    nucamp: {
      institution: "Nucamp",
      credential:
        "Certification in Backend Development with Python, SQL, and DevOps",
      startDate: "2021-08-03",
      endDate: "2021-11-20",
      description:
        "Boot camp course in modern backend Web development with the Python language, PostgreSQL, Flask, Django, and DevOps deployments to Microsoft Azure, Google Cloud, and Amazon AWS.",
      competencies: [
        "Python",
        "Flask",
        "PostgreSQL",
        "Microsoft Azure",
        "Git",
        "CI/CD",
        "Visual Studio Code",
      ],
    },
    tcatk: {
      institution: "Tennessee College of Applied Technology",
      credential: "Diploma in Medical Office Information Technology",
      startDate: "2018-05-01",
      endDate: "2018-08-24",
      description:
        "Broad-ranging instruction in concepts and procedures of the medical billing office, including reimbursement principles, medical terms, medical coding, and practice-management software.",
      competencies: [
        "Medical coding",
        "Medical billing",
        "Microsoft Word",
        "Microsoft Excel",
        "Medical terminology",
        "Teamwork",
      ],
    },
    utk: {
      institution: "University of Tennessee",
      credential: "Bachelor of Science in Computer Science",
      startDate: "2006-08-20",
      endDate: "2011-05-13",
      description:
        "Practical and theoretical instruction in computer science, including systems programming, object-oriented systems, graphical user interfaces, technical writing, and theoretical computer science, combined with traditional liberal-arts curriculum.",
      competencies: [
        "C++",
        "Java",
        "Technical writing",
        "Teamwork",
        "Public speaking",
        "LaTeX",
        "OpenGL",
        "Subversion",
        "Emacs",
      ],
    },
  },
  experience: {
    upaAR: {
      title: "Accounts Receivable Representative",
      company: "University Physicians' Association",
      startDate: "2019-11-18",
      endDate: null,
      duties: [
        "Averaged 30 claims per day with 98% accuracy.",
        "90% remote since March 2020.",
        "Trained other representatives on writing appeal letters.",
        "Responsible for reducing outstanding claim volume by 50% within one quarter.",
        "Recognized as my team's point of knowledge for questions on diagnosis and procedure coding.",
        "Submitted more than 300 appeals to all major payers, overturning 70% of denials.",
      ],
      competencies: [
        "Telecommuting",
        "Medical coding",
        "Medical billing",
        "Writing",
        "Teamwork",
        "Time management",
        "Detail orientation",
        "Microsoft Excel",
        "Microsoft Word",
        "Microsoft Access",
      ],
    },
    upaBilling: {
      title: "Medical Billing Representative",
      company: "University Physicians' Association",
      startDate: "2018-09-10",
      endDate: "2019-11-15",
      duties: [
        "Averaged 1,200 line items per day with 100% accuracy.",
        "Worked in a diverse array of medical specialties including surgery, oncology, and public health.",
        "Made approved edits in accordance with company and payer guidelines.",
        "Contributed to a team responsible for reviewing more than 8,000 line items per day.",
      ],
      competencies: [
        "Medical billing",
        "Medical coding",
        "Data entry",
        "Time management",
        "Teamwork",
        "Microsoft Excel",
        "Detail orientation",
      ],
    },
    wci: {
      title: "Office Cleaner",
      company: "We Clean It CJS",
      startDate: "2006-08-01",
      endDate: "2018-09-09",
      duties: [
        "Cleaned a total of 50,000 sq ft of office space per night.",
        "Achieved a complaint rate of 0% for 7 consecutive years.",
        "Created the company website using WordPress and the <abbr title='Linux, nginx, MariaDB, and PHP'>LEMP</abbr> stack.",
      ],
      competencies: [
        "Time management",
        "Teamwork",
        "Detail orientation",
        "WordPress",
        "Linux",
        "nginx",
      ],
    },
  },
};

export default resumeData;
