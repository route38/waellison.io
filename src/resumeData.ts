export type ResumeData = {
  summary: string,
  skills: SkillsSection,
  education: EducationSection,
  experience: ExperienceSection
};

export type SkillsSection = {
  [key: string]: string[]
};

export type EducationSection = {
  [key: string]: InstitutionType
};

export type InstitutionType = {
  institution: string,
  credential: string,
  startDate: string,
  endDate?: string,
  description: string,
  competencies: string[]
};

export type ExperienceSection = {
  [key: string]: JobType
};

export type JobType = {
  title: string,
  company: string,
  startDate: string,
  endDate?: string,
  duties: string[],
  competencies: string[]
};

export const resumeData: ResumeData = {
  summary:
    "I am a passionate, skilled software developer with a strong background in web development and three years' experience in healthcare revenue cycle management.  My skills include programming in Python and JavaScript with the Flask and React frameworks, PostgreSQL databases, and REST APIs.  I am also skilled at developing for the cloud with Docker, Kubernetes, and Microsoft Azure.  I have recognized skills in analysis, problem solving, training and coaching, and written and oral communication.",
  skills: {
    DevOps: ["azure", "docker", "git", "linux", "continuousIntegration"],
    Libraries: ["flask", "jQuery", "reactjs", "restApi", "postgreSQL"],
    Languages: ["python", "javaScript", "ceePlusPlus", "english", "french"],
  },
  education: {
    fcc: {
      institution: "freeCodeCamp",
      credential: "Certifications in Responsive Web Design and JavaScript Algorithms and Data Structures",
      startDate: "2022-03-15",
      endDate: "2022-04-30",
      description:
        "Self-paced boot camp-style curriculum in major libraries used for frontend Web development, including jQuery, React, Redux, and Bootstrap.",
      competencies: [
        "JavaScript",
        "React.js",
        "Redux",
        "HTML",
        "CSS",
        "Bootstrap",
      ],
    },
    algoexpert: {
        institution: "AlgoExpert",
        credential: "Certificate of Completion",
        startDate: "2022-01-14",
        endDate: "2022-04-22",
        description:
          "Self-paced curriculum in algorithms and data structures, including linked lists, arrays, graphs, trees, strings, recursion, and dynamic programming.",
        competencies: [
          "Python",
          "TypeScript",
          "Algorithms",
          "Data structures"
        ]
    },
    nucamp: {
      institution: "Nucamp",
      credential:
        "Certification in Backend Development with Python, SQL, and DevOps",
      startDate: "2021-08-03",
      endDate: "2021-11-20",
      description:
        "<p>Boot camp course in modern backend Web development with the Python language, PostgreSQL, Flask, Django, and DevOps deployments to Microsoft Azure, Google Cloud, and Amazon AWS.</p><p>Offered coaching and assistance to classmates throughout the course.</p>",
      competencies: [
        "Python",
        "Flask",
        "PostgreSQL",
        "Microsoft Azure",
        "Git",
        "CI/CD",
        "Visual Studio Code",
      ],
    },
    tcatk: {
      institution: "Tennessee College of Applied Technology",
      credential: "Diploma in Medical Office Information Technology",
      startDate: "2018-05-01",
      endDate: "2018-08-24",
      description:
        "<p>Broad-ranging instruction in concepts and procedures of the medical billing office, including reimbursement principles, medical terms, medical coding, and practice-management software.</p><p>Offered coaching and assistance to classmates throughout the course.</p>",
      competencies: [
        "Medical coding",
        "Medical billing",
        "Microsoft Word",
        "Microsoft Excel",
        "Medical terminology",
        "Teamwork",
      ],
    },
    utk: {
      institution: "University of Tennessee",
      credential: "Bachelor of Science in Computer Science",
      startDate: "2006-08-20",
      endDate: "2011-05-13",
      description:
        "Practical and theoretical instruction in computer science, including systems programming, object-oriented systems, graphical user interfaces, technical writing, and theoretical computer science, combined with traditional liberal-arts curriculum.",
      competencies: [
        "C++",
        "Java",
        "Technical writing",
        "Teamwork",
        "Public speaking",
        "LaTeX",
        "OpenGL",
        "Subversion",
        "Emacs",
      ],
    },
  },
  experience: {
    upaAR: {
      title: "Accounts Receivable Representative",
      company: "University Physicians' Association",
      startDate: "2019-11-18",
      endDate: undefined,
      duties: [
        "Collaborated with a 10-member cross-functional team that reduced outstanding claims by 40% quarter-over-quarter, reducing A/R days by 33%",
        "Evaluated and resolved an average of 35 claims per day with 98% accuracy",
        "Assessed denied claims and reviewed for appeal processes",
        "Drafted appeal letters to all major carriers, overturning 70% of denials",
        "Recognized as my team's point of knowledge for questions on diagnosis and procedure coding",
        "Guided and trained personnel within a 55-member department, training on coding rules, standard workflows, software usage, and appeal letter origination",
      ],
      competencies: [
        "Telecommuting",
        "Medical coding",
        "Medical billing",
        "Writing",
        "Teamwork",
        "Time management",
        "Detail orientation",
        "Microsoft Excel",
        "Microsoft Word",
        "Microsoft Access",
      ],
    },
    upaBilling: {
      title: "Medical Billing Representative",
      company: "University Physicians' Association",
      startDate: "2018-09-10",
      endDate: "2019-11-15",
      duties: [
        "Partnered with an established 10-member team to review more than 8,000 outbound line items per day",
        "Reviewed 1,200 line items per day with 100% accuracy",
        "Increased clean claim rate by 5% on average by practice, reducing A/R overhead and turnaround time",
        "Analyzed an average of $100,000 of charges per day for inaccuracies",
        "Proposed workflow alterations which led to 10% faster turnaround of paper charge batches"
      ],
      competencies: [
        "Medical billing",
        "Medical coding",
        "Data entry",
        "Time management",
        "Teamwork",
        "Microsoft Excel",
        "Detail orientation",
      ],
    },
    wci: {
      title: "Office Cleaner",
      company: "We Clean It CJS",
      startDate: "2006-08-01",
      endDate: "2018-09-09",
      duties: [
        "Supervised a two-person cleaning crew maintaining 50,000 square feet per night",
        "Led review initiatives of cleaning and training protocols, increasing productivity by 33%",
        "Redeveloped internal tracking systems to reduce lead time on bids by 50%",
        "Implemented a company website that doubled inbound sales calls",
      ],
      competencies: [
        "Time management",
        "Teamwork",
        "Detail orientation",
        "WordPress",
        "Linux",
        "nginx",
      ],
    },
  },
};
