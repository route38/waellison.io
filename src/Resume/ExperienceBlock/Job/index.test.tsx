import React from "react";
import renderer from "react-test-renderer";
import Job from "./index";
import { resumeData } from "../../../resumeData";

it("renders correct job component", () => {
  const aJob = resumeData.experience.upaAR;
  const component = renderer.create(<Job {...aJob} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
