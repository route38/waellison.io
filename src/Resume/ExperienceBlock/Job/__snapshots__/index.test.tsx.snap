// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correct job component 1`] = `
<div
  className="event"
>
  <h3>
    Accounts Receivable Representative
  </h3>
  <p>
    2019
     to 
    present
     at 
    University Physicians' Association
  </p>
  <ul>
    <li
      dangerouslySetInnerHTML={
        Object {
          "__html": "Averaged 30 claims per day with 98% accuracy.",
        }
      }
    />
    <li
      dangerouslySetInnerHTML={
        Object {
          "__html": "90% remote since March 2020.",
        }
      }
    />
    <li
      dangerouslySetInnerHTML={
        Object {
          "__html": "Trained other representatives on writing appeal letters.",
        }
      }
    />
    <li
      dangerouslySetInnerHTML={
        Object {
          "__html": "Responsible for reducing outstanding claim volume by 50% within one quarter.",
        }
      }
    />
    <li
      dangerouslySetInnerHTML={
        Object {
          "__html": "Recognized as my team's point of knowledge for questions on diagnosis and procedure coding.",
        }
      }
    />
    <li
      dangerouslySetInnerHTML={
        Object {
          "__html": "Submitted more than 300 appeals to all major payers, overturning 70% of denials.",
        }
      }
    />
  </ul>
  <ul
    className="skillsUsed"
  >
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Telecommuting",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Medical coding",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Medical billing",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Writing",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Teamwork",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Time management",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Detail orientation",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Microsoft Excel",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Microsoft Word",
        }
      }
    />
    <li
      className="skillTag"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Microsoft Access",
        }
      }
    />
  </ul>
</div>
`;
